src/app/common/directives/oc-match-field/oc-match-field.js

```diff
@@ -0,0 +1,18 @@
+angular.module('orderCloud')
+    .directive('ocMatchField', OrderCloudMatchFieldDirective)
+;
+
+function OrderCloudMatchFieldDirective() {
+    return {
+        scope: {
+            ocMatchField: '='
+        },
+        restrict: 'A',
+        require: 'ngModel',
+        link: function(scope, element, attr, ngModelCtrl) {
+            ngModelCtrl.$validators.ocMatch = function(val) {
+                return val === scope.ocMatchField;
+            };
+        }
+    };
+}
```

Comment:
Personally I like the way this is structured better than the confirmPassword directive.  It's a lot less code and it actually creates a $ngModelCtrl validator.

I don't believe the confirmPassword directive is being used anywhere in the app so maybe we can create a task to remove that.